backend/reviews/admin.py

```diff
@@ -298,6 +308,33 @@ def _review_grants_recap_view(self, request, review_session):
                         conference_id=review_session.conference_id,
                     )
                 ),
+                ranking=Coalesce(
+                    Subquery(
+                        Submission.objects.filter(
+                            speaker_id=OuterRef("user_id"),
+                            conference_id=OuterRef("conference_id"),
+                        )
+                        .annotate(
+                            ranking=StringAgg(
+                                Concat(
```

Comment:
TODO: improve this, if someone has sent more than one submission, so I would separate each submission ...